src/engine/strat_engine/backstore/crypt.rs

```diff
@@ -265,14 +265,14 @@ impl CryptInitializer {
 
         // Initialize stratis token
         log_on_failure!(
-            device.token_handle().json_set(
-                Some(STRATIS_TOKEN_ID),
+            device.token_handle().json_set(TokenInput::ReplaceToken(
```

Comment:
This change seems correct to me, but I don't understand why it was able to compile without the change, i.e., with just the previous commit.

Reply:
I don't understand either. I wondered the same thing.

Reply:
I found the answer from looking at the git history. For some reason, libcryptsetup-rs doesn't warn us about when a branch is out of sync with master so the branch that I ran CI on didn't include the change in the PR that added `TokenInput` but master did when I released it. You can see this happening visually if you do `git log --graph --online`. However, this should now be resolved. The only potential outstanding issue is that I don't know how to get Github to check if the PR branch is out of sync with master. Do you know how to do this?

Reply:
After doing some digging, I found the Github settings to enforce this and this should be resolved moving forward.

Reply:
> After doing some digging, I found the Github settings to enforce this and this should be resolved moving forward.

Ok!